package.json

```diff
@@ -25,5 +25,8 @@
     "babel-eslint": "^4.1.7",
     "eslint": "^0.22.1",
     "mocha": "^2.2.5"
+  },
+  "peerDependencies": {
+  	"react": "^0.14.0 || ^15.0.0"
```

Comment:
Will this work with react 0.13 as well? This library was first published when react 0.13 was the latest, so it would be good if merging this doesn't make older projects incompatible.